src/plugins/vis_type_timeseries/common/calculate_label.ts

```diff
@@ -42,43 +44,42 @@ export const calculateLabel = (
     return metric.alias;
   }
 
-  if (metric.type === 'count') {
-    return i18n.translate('visTypeTimeseries.calculateLabel.countLabel', {
-      defaultMessage: 'Count',
-    });
-  }
-  if (metric.type === 'calculation') {
-    return i18n.translate('visTypeTimeseries.calculateLabel.bucketScriptsLabel', {
-      defaultMessage: 'Bucket Script',
-    });
-  }
-  if (metric.type === 'math') {
-    return i18n.translate('visTypeTimeseries.calculateLabel.mathLabel', { defaultMessage: 'Math' });
-  }
-  if (metric.type === 'series_agg') {
-    return i18n.translate('visTypeTimeseries.calculateLabel.seriesAggLabel', {
-      defaultMessage: 'Series Agg ({metricFunction})',
-      values: { metricFunction: metric.function },
-    });
-  }
-  if (metric.type === 'filter_ratio') {
-    return i18n.translate('visTypeTimeseries.calculateLabel.filterRatioLabel', {
-      defaultMessage: 'Filter Ratio',
-    });
-  }
-  if (metric.type === 'positive_rate') {
-    return i18n.translate('visTypeTimeseries.calculateLabel.positiveRateLabel', {
-      defaultMessage: 'Counter Rate of {field}',
-      values: { field: extractFieldLabel(fields, metric.field!, isThrowErrorOnFieldNotFound) },
-    });
-  }
-  if (metric.type === 'static') {
-    return i18n.translate('visTypeTimeseries.calculateLabel.staticValueLabel', {
-      defaultMessage: 'Static Value of {metricValue}',
-      values: { metricValue: metric.value },
-    });
+  switch (metric.type) {
+    case METRIC_TYPES.COUNT:
+      return i18n.translate('visTypeTimeseries.calculateLabel.countLabel', {
```

Comment:
we can think about it later. I don't like that duplicates. At least we can remove it for base aggregations. Not critical for that PR